lib/src/parsers/__spec-examples__/path-params.ts

```diff
@@ -11,6 +12,44 @@ class PathParamsClass {
       /** property description */
       "property-with-description": string;
       arrayProperty: string[];
+      /** property-example description
+       * @example property-example
+       * property-example-value
+       *  */
+      "property-with-example": string;
+      /** property-two-examples description
+       * @example property-example-one
+       * 123
+       * @example property-example-two
+       * 456
+       * */
+      "property-with-examples": Integer;
+    },
+    @pathParams
+    paramsWithEmptyExample: {
+      /**@example */
+      property: string;
+    },
+    @pathParams
+    paramsWithDuplicateExampleName: {
+      /**@example name
+       * 123
+       * @example name
+       * 456
+       */
+      property: string;
```

Comment:
```suggestion
      /**@example name
       * "123"
       * @example name
       * "456"
       */
      property: string;
```

To ensure forwards compatibility with object based examples, strings should be explicit

Reply:
What if someone wants to specify a number example?

Reply:
Number examples for `string` types would be numbers in quotes i.e. `"123"`. Number examples for `number` types will be the number without quotes i.e. `123`. Some further examples:
```ts
/**
 * @example string-example
 * "myString"
 *
 * @example string-number-example
 * "123"
 */
stringProp: String;

/**
 * @example number-example
 * 123
 */
intProp: Integer;

/**
 * @example object-example
 * {
 *   "stringAttr": "myString",
 *   "intAttr": 123
 * }
 */
objectProp: {
  stringAttr: String;
  intAttr: Integer;
}
```

Reply:
ah you're right, mistook it for the int example